BikeCSV/src/main/scala/by/itechart/Main.scala

```diff
@@ -1,124 +0,0 @@
-    table.foreach(line => col = line(Column.eleventh.id).replaceAll("\"", "") :: col)
-    col.distinct
-  }
-
-  //task_5
-  def usageStats(): Unit = {
-    val table = tableReader()
-    val strPath = "/usage-stats.csv"
-    val groupMonth = table.groupBy(line => convertToDate(line(Column.first.id)).getMonth)
-
-    val CSVFields = Array("January", "February", "March", "April", "May",
-      "June", "July", "August", "September", "October", "November", "December")
-
-    var listOfRecords = new ListBuffer[Array[String]]()
-    listOfRecords += CSVFields
-    listOfRecords += Array(groupMonth.mapValues(_.size)(Month.zero.id).toString, groupMonth.mapValues(_.size)(Month.first.id).toString,
-      groupMonth.mapValues(_.size)(Month.second.id).toString, groupMonth.mapValues(_.size)(Month.third.id).toString,
-      groupMonth.mapValues(_.size)(Month.fourth.id).toString, groupMonth.mapValues(_.size)(Month.fifth.id).toString,
-      groupMonth.mapValues(_.size)(Month.sixth.id).toString, groupMonth.mapValues(_.size)(Month.seventh.id).toString,
-      groupMonth.mapValues(_.size)(Month.eighth.id).toString, groupMonth.mapValues(_.size)(Month.ninth.id).toString,
-      groupMonth.mapValues(_.size)(Month.tenth.id).toString, groupMonth.mapValues(_.size)(Month.eleventh.id).toString)
-
-    tableWriter(strPath, listOfRecords)
-  }
-
-
-  //task_6
-  def bikeStats(): Unit = {
-    val strPath = "/bike-stats.csv"
-    val CSVFields = Array("bikeId", "number of trip" ,"time using")
-
-    val tableCSV = tableReader
-    val groupBikeId = tableCSV
-      .map(strArr => (strArr(Column.eleventh.id),
-        TimeUnit.MINUTES.convert(convertToDate(strArr(Column.second.id)).getTime -
-          convertToDate(strArr(Column.first.id)).getTime, TimeUnit.MILLISECONDS)))
-      .groupBy(_._1)
-
-    val numTrip = groupBikeId.view.mapValues(_.size).toList
-    val timeUsing = groupBikeId.view.mapValues(_.map(_._2).sum).toList
-    val bikeStatsList = (numTrip.map(line => line._1), numTrip.map(line => line._2), timeUsing.map(line => line._2)).zipped.toArray
-
-    var listOfRecords = new ListBuffer[Array[String]]()
-    listOfRecords += CSVFields
-    for(l <- bikeStatsList){
-      listOfRecords += Array(l._1.replaceAll("\"", ""), l._2.toString, l._3.toString)
-    }
-
-    tableWriter(strPath, listOfRecords)
-  }
-
-  private def tableReader(): List[Array[String]] = {
-    Source.fromURL(getClass.getResource("/sources/201608-citibike-tripdata.csv"))
-      .getLines()
-      .map(_.split(","))
-      .toList
-      .drop(1)
-  }
-
-  private def tableWriter(strPath: String, listOfRecords: ListBuffer[Array[String]]): Unit = {
-    val out = new BufferedWriter(new FileWriter("D:/Scala/WorkSpace/ScalaIntership/TripData/src/main/resources/sources" + strPath))
-    new CSVWriter(out).writeAll(listOfRecords.asJava)
-    out.close()
-  }
-
-  private def convertToDate(string: String): Date = {
-    val inputFormat = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss")
```

Comment:
Does that sound like something related to Date parsing logic and not the Main.scala?
Could that be moved to some other class? 
= Single Responsibility Principle